lib/mongoose_push_web/endpoint.ex

```diff
@@ -0,0 +1,41 @@
+defmodule MongoosePushWeb.Endpoint do
+  use Phoenix.Endpoint, otp_app: :mongoose_push
+
+  # The session will be stored in the cookie and signed,
+  # this means its contents can be read but not tampered with.
+  # Set :encryption_salt if you would also like to encrypt it.
+  @session_options [
+    store: :cookie,
+    key: "_mongoose_push_key",
+    signing_salt: "URIBGSwL"
+  ]
+
+  # Serve at "/" the static files from "priv/static" directory.
+  #
+  # You should set gzip to true if you are running phx.digest
+  # when deploying your static files in production.
+  plug Plug.Static,
+    at: "/",
+    from: :mongoose_push,
+    gzip: false,
+    only: ~w(css fonts images js favicon.ico robots.txt)
```

Comment:
This one is also not needed. We'll probably don't have static files to serve.

Reply:
@rslota what do you think?

Reply:
Yeah, also left comment for that :)